lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1106,6 +1106,16 @@ sub save_user {
            my $template = LedgerSMB::Template->new_UI(
                $request,
                template => 'setup/new_user',
+               hiddens  => {
+                                employeenumber => $request->{employeenumber},
+                                 salutation_id => $request->{salutation_id},
+                                         perms => $request->{perms},
+                                           ssn => $request->{ssn},
+                                    first_name => $request->{first_name},
+                                     last_name => $request->{last_name},
+                                           dob => $request->{dob},
+                                      username => $request->{username},
+                           },
```

Comment:
What problem does this solve? Note that this isn't feeding the default values "back" into the editboxes.

Reply:
It certainly is

Reply:
I'm sorry, that's too terse for me. It is what? (or isn't what?)

Reply:
This does feed the default values in the dialog, it sets the hiddens and I use them in the form.

Reply:
Ok. If that's the case, then where do you do that? I only see you use values of the variables `ssn`, `dob`, `last_name`, etc. (which definitely don't come from `hiddens`).